src/main/resources/com/cockroachlabs/university/javatransactions/dao/ItemDao/updateItemQuantity.sql

```diff
@@ -0,0 +1,9 @@
+--
+-- updateItemQuantity
+--
+
+UPDATE items
+   SET quantity = quantity - :amount
+ WHERE item_id = :item_id
+   ;
```

Comment:
For production, we should move the commented line below up here.
```suggestion
  AND quantity >= :amount
;
```